nmdc_schema/identify_workflow_duplicates_emp500.py

```diff
@@ -0,0 +1,214 @@
+import json
+import pymongo
+import os
+from pymongo import MongoClient
+from pymongo.errors import ConnectionFailure
+from dotenv import load_dotenv
+import urllib
+
+# this code deletes multiple workflow records from EMP500 which have different base IDs with a .1 version.
```

Comment:
This should be more accurate - this is generating a .json data file that can be used to delete these records by submitting it to the appropriate endpoint

Reply:
updated